### ISSUE TITLE
All Jobs table headings don't deal well with narrow page

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-web#345]
When one makes the all job page narrow, the headings merge. Even on normal size, some overlap:

<img width="1334" alt="Screenshot 2023-11-24 at 16 53 02" src="https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/612fa2ce-0ab3-40a2-bb90-50e3ca00b5c9">
<img width="1088" alt="Screenshot 2023-11-24 at 16 52 57" src="https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/2939a5df-726d-4619-87dd-6941f3af85d7">

2 suggestions:
1. Can we make the heading optional multi-line (same as e.g. the long names of workflows), if it doesn't fit otherwise? Or crop it?
2. Do we want to drop the ID from the table view? It's still available under info. It's sometimes useful, but not a core things users need to see

**Comment by zonia3000** [comment]
> 1. Can we make the heading optional multi-line (same as e.g. the long names of workflows), if it doesn't fit otherwise? Or crop it?

If we crop or make multi-line the table headers we obtain very small cells that are not very good for readability.

It is possible to avoid the overlap of the labels defining a minimum width for table headers. In this way the table can be scrolled horizontally on small screens. There are 2 approaches to handle the horizontal scroll.

1. Simply let the table grow horizontally. In this way the page is larger than the window and the user can scroll the table using the main browser horizontal scrollbar:

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/03436994-b6de-4cc3-b0fb-8ed3386875ee)

This is the most usable way to go when we have very long tables, since the horizontal scrollbar is always available. The drawback is that the top menu bar is truncated at the widow width and the page looks asymmetrical.

You can test this using the commit 01f95e211886443efb22d0d8e1ca7d66375592ac.

2. We can wrap the table in a container that uses the Bootstrap `table-responsive` CSS class:

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/8c440faf-c8fa-44a0-8835-23a7c3ae4585)

In this case the page still has the size of the window but the table is wrapped in a container having its own horizontal scrollbar. This produces a nicer, symmetric page, but to reach the scrollbar one has to go down at the end of the table and it can be quite annoying if the table is long, so I would not suggest it on this case.
 
> 2. Do we want to drop the ID from the table view? It's still available under info. It's sometimes useful, but not a core things users need to see

Done on same commit mentioned above.

**Comment by jluethi** [comment]
Thanks! 

One thing we should change in this case though is the order of columns: A user should always see status & Options first, because those they want to interact with. If we order them Status, Options, Start, End, Project, Workflow, Datasets, that would be solved.


I'm a bit torn between option 1 & 2 above. I just tested the UZH Fractal server from my phone and saw that this already uses a version with Option 1 & multi-line column titles.*
![IMG_4902](https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/063c2fe5-f5c3-4fb0-80e4-87ea96dcf8b3)


So this table is already somewhat responsive and the more we can use that responsiveness, the better. I'm not sure though what triggers this. And checking Fractal web 0.6.3, that also behaves as expected: 
<img width="1138" alt="Screenshot 2023-11-27 at 11 51 05" src="https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/84088bc7-24a3-445b-8be4-971cf08d49ca">
<img width="612" alt="Screenshot 2023-11-27 at 11 54 13" src="https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/a810a29b-bf3a-4bc1-81d1-e545215a9bc8">

Thus, making the table a bit more compact and making the headings multi-line when needed.

I don't know whether option 1 or 2 would be better on mobile. But on the desktop experience (that we are optimizing for), I like the behavior in 0.6.3 better. It basically squeezes the table a bit (with multi-line titles) and at some point goes to option 1 from above.


* Yay that the whole thing works from iOS safari!!

**Comment by jluethi** [comment]
tl;dr: Some table responsiveness is nice. We used to have some multi-line headings in some cases.
When everything is too small, going with option 1 or 2 from above seems fine

**Comment by zonia3000** [comment]
Ok, I ended up in both changing columns order and making the header multi-line.

ref 6e655057a34ea0a7001d65f8e44e7167d4174b37

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/5797fea2-0f5c-4018-981f-13bf2cdb00c0)

**Comment by jluethi** [comment]
Great! This looks good to me!

Will take some getting used to having status & options on the left. But I think it's a good idea :)
